sdk/js/src/Client.ts

```diff
@@ -115,6 +125,9 @@ export class DVCClient implements Client {
                 .catch((err) => onInitialized(err))
             return
         }
+        if (this.isConfigCached) {
+            this.logger.info("Initialized with a cached config")
```

Comment:
by the time the client has initialized we would have fetched a config and set this back to false, we should log after setting the cached config on the client and triggering the onUpdate callbacks

Reply:
how to trigger the onUpdate callbacks?